account_financial_report/report/templates/general_ledger.xml

```diff
@@ -378,6 +381,10 @@
                         </span>
                     </div>
                     <t t-if="line.currency_id">
+                        <!--## currency_name-->
```

Comment:
@hveficent Why add this one, isn't it easier to have the monetary widget with currency_id,